go/dhcp/config.go

```diff
@@ -270,6 +283,19 @@ func (d *Interfaces) readConfig() {
 						DHCPScope.xid = xid
 
 						initiaLease(&DHCPScope)
+
+						excludeIPs := IPsFromRange(ConfNet.IpReserved)
```

Comment:
This is a copy/paste of the code above, externalize this to a function to DRY up the code